src/sql/src/plan/statement/scl.rs

```diff
@@ -43,16 +46,56 @@ pub fn plan_set_variable(
     SetVariableStatement {
         local,
         variable,
-        value,
+        to,
     }: SetVariableStatement,
 ) -> Result<Plan, PlanError> {
+    let value = plan_set_variable_to(to)?;
     Ok(Plan::SetVariable(SetVariablePlan {
-        name: variable.to_string(),
+        name: variable.into_string(),
         value,
         local,
     }))
 }
 
+/// Split a single string into what would be parsed by `SET v TO <string>` into
+/// multpile values.
+pub fn parse_set_variable_value(value: &str) -> Result<Vec<String>, PlanError> {
```

Comment:
I think the `plan` module is a bit of an awkward place for this function, because as far as I can tell it's never used during planning. Not a huge deal though, if this is just the easiest place to put it for dependency reasons.

Reply:
I also really don't like it here, but was not sure where to put it. This was the least bad spot I could find? One choice is the parser crate, but we try real hard to keep sql logic out of the parser.

Reply:
Actually: this is only called from adapter, so it can move there.